packages/tangle/data_payload.go

```diff
@@ -0,0 +1,125 @@
+package tangle
+
+import (
+	"fmt"
+
+	"github.com/iotaledger/hive.go/marshalutil"
+	"github.com/iotaledger/hive.go/stringify"
+)
+
+const (
+	// ObjectName defines the name of the data object.
+	ObjectName = "data"
```

Comment:
I think for now it's fine to name it `Object` but eventually we need to decide to stick to either `Object` or `Payload` and change it across the whole codebase. Even the file is called `data_payload.go`.